CWE/localisation/countries.csv

```diff
@@ -280,4 +280,6 @@ RBU_absolute_monarchy;The Kingdom of Russia and Belarus;;;;;;;;;x
 RBU_hms_government;The Kingdom of Russia and Belarus;;;;;;;;;x
 RBU_hms_government1;The Kingdom of Russia and Belarus;;;;;;;;;x
 RDA;North Afghanistan;;;;;;;;;x
-RDA_ADJ;Afghan;;;;;;;;;x
\ No newline at end of file
+RDA_ADJ;Afghan;;;;;;;;;x
+TRB;Zone B;;;;;;;;;x
```

Comment:
Please give a more specific name rather than just “Zone B” can call it East Trieste or something but please no generic names.